valhalla/baldr/rapidjson_utils.h

```diff
@@ -18,7 +18,8 @@
 #define RAPIDJSON_ASSERT(x)                                                                          \
   if (!(x))                                                                                          \
   throw std::logic_error(RAPIDJSON_STRINGIFY(x))
-// Because we now throw exceptions, we need to turn of RAPIDJSON_NOEXCEPT
+// Because we now throw exceptions, we need to turn off RAPIDJSON_NOEXCEPT
+#undef RAPIDJSON_HAS_CXX11_NOEXCEPT
```

Comment:
can't remember, but think this was to get rid of a warning

Reply:
I would have expected you to get a redefinition below can you post what happens when you remove this change

Reply:
On gcc 11.1.0 I get this:
```
[build] [95/123  69% :: 257.071] Building CXX object src/tyr/CMakeFiles/valhalla-tyr.dir/serializers.cc.o
[build] In file included from ../src/tyr/serializers.cc:15:
[build] ../valhalla/baldr/rapidjson_utils.h:22: warning: "RAPIDJSON_HAS_CXX11_NOEXCEPT" redefined
[build]    22 | #define RAPIDJSON_HAS_CXX11_NOEXCEPT 0
[build]       | 
[build] In file included from ../third_party/rapidjson/include/rapidjson/allocators.h:18,
[build]                  from ../src/tyr/serializers.cc:5:
[build] ../third_party/rapidjson/include/rapidjson/rapidjson.h:562: note: this is the location of the previous definition
[build]   562 | #define RAPIDJSON_HAS_CXX11_NOEXCEPT 1
[build]       | 
[build] In file included from ../src/tyr/serializers.cc:15:
[build] ../valhalla/baldr/rapidjson_utils.h:24: warning: "RAPIDJSON_HAS_STDSTRING" redefined
[build]    24 | #define RAPIDJSON_HAS_STDSTRING 1
[build]       | 
[build] In file included from ../third_party/rapidjson/include/rapidjson/allocators.h:18,
[build]                  from ../src/tyr/serializers.cc:5:
[build] ../third_party/rapidjson/include/rapidjson/rapidjson.h:134: note: this is the location of the previous definition
[build]   134 | #define RAPIDJSON_HAS_STDSTRING 0 // no std::string support by default
```

Was just sick of seeing this I guess, but open to other ways of avoiding that warning

Reply:
any preference here?

Reply:
you put the allocators include in here but forgot to remove this `undef`

```suggestion
```